### PR TITLE
docs: improve consumability and accuracy of fscloud doc

### PR DESCRIPTION
**Pull request by vburckhardt**
### Description

Improve consumability and accuracy of fscloud doc

Addresses https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/issues/48

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] The PR references a GitHub issue.
- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

File: profiles/fscloud/README.md [hunk 1]
```diff
@@ -1,6 +1,59 @@
...
+This module is a version of the [parent root module](../../) that comes with pre-defined defaults that meet the relevant controls from the [IBM Cloud Framework for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about).
 
-## Note: If the Allowlist is not set and no Context Based Restriction(CBR) rules are not passed, you must configure Context Based Restrictions externally to be compliant.
+See the accompanying [fscloud example](../../examples/fscloud/) leveraging this module.
+
+The defaults coming with this module have been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and the Financial Cloud profile for [Security and Compliance Center](https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-getting-started). They meet all applicable goals.
+
+:exclamation: The financial services framework mandates the application of an inbound network-based allow-list in front of the ICD PostGresql instance. This can be achieved in multiple ways:
```
Comment (SirSpidey) [inline profiles/fscloud/README.md:9]:
```suggestion
:exclamation: **Important:** The Financial Services framework mandates the application of an inbound network-based allowlist in front of the ICD PostgreSQL instance. You can comply in multiple ways:
```

**Review by SirSpidey — CHANGES_REQUESTED** [review]
@vburckhardt A bunch of style suggestions. Let me know if it's too hard to commit these suggestions and I can make a PR to this PR instead.

File: examples/fscloud/main.tf [hunk 2]
```diff
@@ -75,7 +75,7 @@ module "cbr_zone" {
 }
 
 ##############################################################################
-# Postgres Instance
+# Postgres Instance (FsCloud Profile)
```
Comment (maheshwarishikha) [inline examples/fscloud/main.tf:78]: It should be FSCloud.

**Comment by SirSpidey** [comment]
@vburckhardt wanted to make sure you saw my comments on this PR. It has been open for a week.

**Comment by SirSpidey** [comment]
@vburckhardt checking in again on this open PR. I see that you made some updates. What is left (besides addressing conflicts)?

**Comment by SirSpidey** [comment]
@vburckhardt Does the `/profiles/fscloud/readme.md` content in #185 cover this PR?

**Comment by ocofaigh** [comment]
Updates included in https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/pull/186

**Pull request closed without merging**